addon/components/paper/text-field.js

```diff
@@ -1,24 +1,33 @@
 import Component     from '@ember/component';
-import { isPresent } from '@ember/utils';
 
 import formFieldOptions from
   'ember-railio-form-components/mixins/form-field-options';
 import { computed, get, set } from '@ember/object';
 
+function focussedOut(event) {
```

Comment:
Maybe rename this to something like `isFocusOutEvent`, to more clearly define what this function does